service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/OrderIssuedEmailContentProvider.java

```diff
@@ -84,6 +85,17 @@ public OrderIssuedEmailContentProvider(@Value("${ccd.ui.base.url}") String uiBas
             .build();
     }
 
+    public Map<String, Object> buildOrderNotificationParameters(final CaseDetails caseDetails,
```

Comment:
maybe this could be named `buildNotificationParametersWithCaseUrlAndAttachedDocument`

and the method above `buildNotificationParametersWithAttachedDocument`
thoughts? might be a bit clearer

Reply:
we can leave as-is, as this can be reused generically. Also, renaming the method method above changes its existing function and leads to more changes thereby bloating the pr.